### PR TITLE
Consistently use 'Capture' verb

### LINKED ISSUE #195: More consistent terminology
## Problem

Right now we use “grab” and “take” within the UI itself, and then non-verby-terms for the .desktop actions. 

## Proposal

We should probably use the term “take” or “capture” consistently in all three places.

## Prior Art

Android 11 just has “Screenshot” as the action in its UI, so that’s not very helpful. I’m not aware what Windows, macOS, Chrome OS, and iOS do.

### PR DESCRIPTION
**Pull request by cassidyjames**
Intends to fix #195. Open to discussion.

### CODE REVIEW

**Comment by danrabbit** [comment]
Hrrmmmm I haven't pulled this yet but my first couple of thoughts are:

Having capture in every label in the UI seems bad. I feel like it's generally not good to be too repetitive, especially when trying to show the differences between options

"Capture screenshot of the whole screen" doesn't seem better than "Screenshot the whole screen"

Perhaps we should lean towards just using "screenshot" as a verb where it makes sense. Our window menu label could be "Screenshot This Window" and that seems pretty clear without being overly verbose 

Reading it back, I also feel like both "Grab pointer" and "Capture pointer" are probably less clear than "Include pointer"

**Comment by fabiozaramella** [comment]
> Having capture in every label in the UI seems bad. I feel like it's generally not good to be too repetitive, especially when trying to show the differences between options

I agree, it would end up being repetitive.

> Perhaps we should lean towards just using "screenshot" as a verb where it makes sense. Our window menu label could be "Screenshot This Window" and that seems pretty clear without being overly verbose

I think it's better to avoid using nouns as verbs when possible, especially when terms like this are not really translatable into other languages. For example "Grab this Window" is translated in italian as "Cattura questa finestra" whereas "Screenshot this Window" can't be translated properly. I mean, we can keep the same translation of course but my concern is that some other new translators (or in other languages) may translate improperly for example using the slang "Screenshottare" (to take a screenshot) in the italian case.

> Reading it back, I also feel like both "Grab pointer" and "Capture pointer" are probably less clear than "Include pointer"

I also agree that "Include pointer" is better.

**Comment by cassidyjames** [comment]
Yeah being repetitive is a problem. But before we were using at least 3 different verbs, so I think we should align on what verb we use in general for taking screenshots and use that consistently. At the same time, we should work to reduce repetitiveness.

**Comment by cassidyjames** [comment]
Okay I like this more. It's more concise in general and should address what you both brought up. Now I guess, do we like "Capture" or "Take" better? Regarding .desktop action names, see: https://github.com/elementary/applications-menu/issues/277 as I think that will also make this work better, as with other apps.

**Review by fabiozaramella — APPROVED** [review]
Not in the UX team and I still want to hear from Daniel, but is approval from me.

File: src/ScreenshotWindow.vala [hunk 1]
```diff
@@ -117,7 +117,7 @@ public class Screenshot.ScreenshotWindow : Hdy.ApplicationWindow {
 
         var delay_spin = new Gtk.SpinButton.with_range (0, 15, 1);
 
-        var take_btn = new Gtk.Button.with_label (_("Take Screenshot"));
+        var take_btn = new Gtk.Button.with_label (_("Capture Screenshot"));
```
Comment (danrabbit) [inline src/ScreenshotWindow.vala:120]: I think I like "take" more here tbh. I'm not sure why but it feels more natural to me for some reason

File: data/screenshot.desktop.in [hunk 2]
```diff
@@ -10,13 +10,13 @@ Keywords=Snapshot;Screenshot;
...
 Exec=io.elementary.screenshot --screen
 
 [Desktop Action WindowShot]
-Name=Screenshot of the Current Window
+Name=Capture the Current Window
 Exec=io.elementary.screenshot --window
 
 [Desktop Action AreaShot]
-Name=Screenshot of a Selected Area
+Name=Select Area to Capture
```
Comment (danrabbit) [inline data/screenshot.desktop.in:21]: Hm, should this instead be
```suggestion
Name=Capture a Selected Area
```

**Review by danrabbit — CHANGES_REQUESTED** [review]
Left a couple inline comments.

Have you tested that actions still appear when searching for "Screenshot" even if their text doesn't contain that word?

**Comment by danrabbit** [comment]
This hasn't moved in a month so I'm gonna kick it to a draft

File: src/ScreenshotWindow.vala (hunk 1 above)
Comment (cassidyjames) [inline src/ScreenshotWindow.vala:120]:
```suggestion
        var take_btn = new Gtk.Button.with_label (_("Take Screenshot"));
```

**Comment by cassidyjames** [comment]
@danrabbit couple of comments addressed.

> Have you tested that actions still appear when searching for "Screenshot" even if their text doesn't contain that word?

As mentioned in https://github.com/elementary/screenshot/issues/196#issuecomment-721924434:

>Regarding .desktop action names, see: https://github.com/elementary/applications-menu/issues/277 as I think that will also make this work better, as with other apps.

Do you consider that a blocker?

**Pull request closed without merging**